### ISSUE TITLE
Better Raspberry Pi instructions

### ISSUE THREAD

**Issue by Lartza** [piface/pifacecad#29]
The current installation instructions for RPi should be rewritten to use a virtual environment instead. Or anything else than sudoing setup.py or pip really

For both security and package management reasons

**Comment by tompreston** [comment]
Agreed, my personal preference is [using `pip install --user`, or better `pipenv`](https://docs.python-guide.org/dev/virtualenvs/).
